audformat/core/utils.py

```diff
@@ -64,18 +65,35 @@ def concat(
     or one column contains ``NaN``.
     If ``overwrite`` is set to ``True``,
     the value of the last object in the list is kept.
+    If ``overwrite`` is set to ``False``,
+    a custom aggregation function can be provided
+    with ``aggregate_function``
+    that converts the overlapping values
+    into a single value.
 
     Args:
         objs: objects
         overwrite: overwrite values where indices overlap
+        aggregate_function: function to aggregate overlapping values.
+            The function gets a :class:`pandas.Series`
+            with overlapping values
+            as input.
+            E.g. set to
+            :func:`numpy.mean`
+            to average the values
+            or to
+            ``lambda row: tuple(row.to_list())``
```

Comment:
I think we can simply do:

```suggestion
            ``lambda row: tuple(row)``
```

and I think I would rewrite to:

```
lambda y: tuple(y)
```

Reply:
Changed to your second suggestion